build/images/scripts/install_cni_chaining

```diff
@@ -1,7 +1,13 @@
 #!/usr/bin/env bash
 
 # Find the cni conf file with lowest name
-cni_conf=$(ls /host/etc/cni/net.d | head -n1)
+while true; do
+  cni_conf=$(ls /host/etc/cni/net.d | head -n1)
+  if [[ ! -z $cni_conf ]]; then
+    break
+  fi
+  sleep 2s
```

Comment:
Could we add some logging? otherwise it would be confusing the initContainer keeps running without any message if the cni conf is not available for a long time.
You may `source logging` and use `log_info` like https://github.com/vmware-tanzu/antrea/blob/master/build/images/scripts/start_ovs#L14